Description: Disable Google APIs
 Google Apis are not in Debian yet.
Author: Kai-Chung Yan
Forwarded: not-needed
--- a/settings.gradle
+++ b/settings.gradle
@@ -30,7 +30,6 @@
 include 'messaging'
 include 'resources'
 include 'resourcesHttp'
-include 'resourcesGcs'
 include 'resourcesSftp'
 include 'plugins'
 include 'scala'
--- a/build.gradle
+++ b/build.gradle
@@ -153,7 +153,7 @@
         'platformJvm', 'languageJvm', 'languageJava', 'languageGroovy', 'languageScala',
         'platformNative', 'platformPlay', 'idePlay', 'languageNative', 'ideNative', 'testingBase',
         'testingNative', 'testingJvm', 'pluginDevelopment', 'pluginUse', 'resourcesHttp',
-        'resourcesSftp', 'resourcesGcs', 'compositeBuilds', 'buildCacheHttp'
+        'resourcesSftp', 'compositeBuilds', 'buildCacheHttp'
     ].collect { project(it) }
     implementationPluginProjects = [
         project('toolingApiBuilders')
